lib/metasploit/framework/login_scanner/redis.rb

```diff
@@ -50,40 +52,49 @@ def attempt_login(credential)
             connect
             select([sock], nil, nil, 0.4)
 
-            command = redis_proto(['AUTH', "#{credential.private}"])
-            sock.put(command)
-            result_options[:proof] = sock.get_once
```

Comment:
Removing this proof will cause issues. First we lose the 'proof' data that we'll persist into the database, and it also causes the module to break later when the proof is shown to the user

Full debug thread: https://github.com/rapid7/metasploit-framework/pull/19283#issuecomment-2191381527

Reply:
Reworked the code to retain the proof and status as originally intended.